02-express-tutorial/10-middleware-options

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const app = express()
+const morgan = require('morgan')
+const logger = require('./logger')
+const authorize = require('./authorize')
+// req => middleware => res
+
+//app.use('/api',logger)//will invoke logger for any route, order matters, if no path argument is provided it's invoked for all paths
+//app.use([logger, authorize])
+//app.use(express.static('./public'))
+
+app.use(morgan('tiny'))
+
+app.get('/',(req,res)=>{//why aren't we passing in req, res in logger
+   res.send('Home')
+})
+
```

Comment:
Because, app.get() call not call middleware by itself you leave the second argument out. If you to call a middleware in this endpoint, you will have to add  like this:
 
app.get('/', middleware, (req,res)=>{//why aren't we passing in req, res in logger
   res.send('Home')
})

Remember the differences between app.use and app.METHOD()  and what they do; { Method in app.METHOD() can be any get/post/patch/delete/put etc.}